code/Player/Player.Confirmation.cs

```diff
@@ -29,6 +29,24 @@ public partial class Player
 	public Player LastSeenPlayer { get; set; }
 	public List<Player> PlayersKilled { get; set; } = new();
 
+	private string _lastWords;
+	private TimeSince _timeSinceLastWords;
+	public string LastWords
+	{
+		get
+		{
+			if ( this.IsAlive() && _timeSinceLastWords < 3 )
```

Comment:
If a player sends a message within the last three seconds of them being alive, they have their last words added to their corpse. 

I don't think the `this.IsAlive()` check is needed and it should be `_timeSinceLastWords > 3`